src/main/resources/db/changelog/init/sql/insert_data.sql

```diff
@@ -4,37 +4,43 @@ VALUES (1, 'USER', current_timestamp, current_timestamp);
-VALUES (4, 'Ukraine', 'Rivne', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city, loc_address_line, loc_created, loc_modified)
+VALUES (2, 'UA', 'Lviv', 'Sichovyh Strilciv, 54', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (5, 'Ukraine', 'Chernigov', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city, loc_address_line, loc_created, loc_modified)
+VALUES (3, 'BY', 'Minsk', 'Independence Avenue, 1', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (6, 'Belarus', 'Minsk', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (4, 'Ukraine', 'Rivne', 'Beregova, 13', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (7, 'Belarus', 'Grodno', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (5, 'Ukraine', 'Chernigov', 'Ivana Bogyna, 6', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (8, 'Belarus', 'Gomel', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (6, 'Belarus', 'Minsk', 'Kirova, 17', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (9, 'Belarus', 'Berestia', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (7, 'Belarus', 'Grodno', 'Kalinina, 3', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (10, 'Belarus', 'Mogilev', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (8, 'Belarus', 'Gomel', 'Artema, 5', current_timestamp, current_timestamp);
 
-INSERT INTO category (cat_id, cat_title, cat_image_url, cat_created, cat_modified)
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (9, 'Belarus', 'Brest', 'Gogolia, 9', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (10, 'Belarus', 'Mogilev', 'Aviatorov, 15', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
 VALUES (1, 'Sports and fitness', 'http://localhost/images/sport_image.jpeg', current_timestamp, current_timestamp);
```

Comment:
country_code and loc_city, loc_address_line are missed here

Reply:
Changed